cms/djangoapps/cms_user_tasks/tasks.py

```diff
@@ -32,6 +32,14 @@ def send_task_complete_email(self, task_name, task_state_text, dest_addr, detail
         'task_status': task_state_text,
         'detail_url': detail_url
     }
+    if olx_validation_text:
+        try:
+            olx_validations = json.loads(olx_validation_text)
+            context['olx_validation_errors'] = True
+            context['error_summary'] = olx_validations.get('error_summary')
+            context['error_report'] = olx_validations.get('error_report')
+        except ValueError:  # includes simplejson.decoder.JSONDecodeError
+            pass
```

Comment:
we should add a log here, just in case it might be handy when we don't get expected error messages to load and Exception is raised.